source/options/GameplaySettingsSubState.hx

```diff
@@ -56,6 +56,13 @@ class GameplaySettingsSubState extends BaseOptionsMenu
 			false);
 		addOption(option);
 
+		var option:Option = new Option('Left Side',
+		'If checked, You play the opposite side. Does not change healthbar or arrow location.',
```

Comment:
not a big error, but, in the rules, uppercases are not used after a comma, only after a full stop punctuation
sorry if i'm disturbing here